extensions/blocks/story/player/slide.js

```diff
@@ -41,17 +42,36 @@ export const Slide = ( {
 		timeout: null,
 	} );
 
+	const playVideoWithFallback = async mediaElement => {
+		try {
+			await mediaElement.play();
+		} catch ( err ) {
+			// try playing again when muted is set
+			setMuted( true );
+		}
+	};
+
+	// Sync muted state with underlying HTMLMediaElement
+	useEffect( () => {
+		if ( isVideo() ) {
+			mediaRef.current.muted = muted;
+			if ( ! muted ) {
+				mediaRef.current.volume = settings.volume;
+			}
+		}
+	}, [ muted ] );
+
 	// Sync playing state with underlying HTMLMediaElement
 	// AJAX loading will pause the video when the video src attribute is modified
 	useEffect( () => {
 		if ( isVideo() ) {
 			if ( currentSlidePlaying ) {
-				mediaRef.current.play();
+				playVideoWithFallback( mediaRef.current );
```

Comment:
I'm seeing an issue where on browsers that need to be muted, it's not possible to unmute the video while it's playing, unless you pause playback first.

I think the cause might be that we're calling this every time the mute button is pressed. If the video is already playing we try to `play()` again, which results in the same error - instead if the video is already playing we shouldn't try `play()` again and just unmute.